src/net.h

```diff
@@ -792,7 +853,7 @@ class CNode
 
     // flood relay
     std::vector<CAddress> vAddrToSend;
-    const std::unique_ptr<CRollingBloomFilter> m_addr_known;
+    std::unique_ptr<CRollingBloomFilter> m_addr_known = nullptr;
```

Comment:
nit: more common style these days is to use braced initialization (although it's not in the style guide).